profiles/common/modules/features/ec_embedded_video/includes/ec_embedded_video.formatters.inc

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * @file
+ * File formatters for Platform needs.
+ */
+
+/**
+ * Implements hook_file_formatter_info_alter().
+ *
+ * It alters file_formatter_info to add the "Wrapper free formatters for the
+ * different Video-sharing services (YouTube, Vimeo, Dailymotion and
+ * AV portal).
+ */
+function ec_embedded_video_file_formatter_info_alter(&$info) {
+  if (module_exists('media_youtube')) {
+    $formatter = $info['media_youtube_video'];
+    $formatter['label'] = t('YouTube Video Wrapper Free');
+    $formatter['description'] = t(
+      'It works like the "YouTube Video" formatter but it does not wrap the 
+      video tags by HTML containers like "div"'
+    );
+    $formatter['view callback'] = '_no_wrapper_youtube_file_formatter_video_view';
+    $info['no_wrapper_media_youtube_video'] = $formatter;
+  }
+
+  if (module_exists('media_vimeo')) {
+    $formatter = $info['media_vimeo_video'];
+    $formatter['label'] = t('Vimeo Video Wrapper Free');
+    $formatter['description'] = t(
+      'It works like the "Vimeo Video" Wrapper but it does not wrap the 
+      video tags by HTML containers like "div"'
+    );
+    $formatter['view callback'] = '_no_wrapper_vimeo_file_formatter_video_view';
+    $info['no_wrapper_media_vimeo_video'] = $formatter;
+  }
+
+  if (module_exists('media_dailymotion')) {
+    $formatter = $info['media_dailymotion_video'];
+    $formatter['label'] = t('Dailymotion Video Wrapper Free');
+    $formatter['description'] = t(
+      'It works like the "Dailymotion Video" Wrapper but it does not wrap the 
+      video tags by HTML containers like "div"'
+    );
+    $formatter['view callback'] = '_no_wrapper_dailymotion_file_formatter_video_view';
+    $info['no_wrapper_media_dailymotion_video'] = $formatter;
+  }
+
+  if (module_exists('media_avportal')) {
+    $formatter = $info['media_avportal_video'];
+    $formatter['label'] = t('AV Portal Video Wrapper Free');
+    $formatter['description'] = t(
+      'It works like the "AV Portal Video" Wrapper but it does not wrap the 
+      video tags by HTML containers like "div"'
+    );
+    $formatter['view callback'] = '_no_wrapper_avportal_file_formatter_video_view';
+    $info['no_wrapper_media_avportal_video'] = $formatter;
+  }
+}
+
+/**
+ * Implements hook_file_formatter_FORMATTER_view().
+ *
+ * It adds the 'no_wrapper' variable value in options of the YouTube video
+ * elements.
+ */
+function _no_wrapper_youtube_file_formatter_video_view($file, $display, $langcode) {
+  $element = media_youtube_file_formatter_video_view($file, $display, $langcode);
+
+  if (!empty($element)) {
```

Comment:
Like this, you do not have the element's attributes injected by the media module. Do not forget here that file filters are not combined; File entity try the first one  and if it is not applicable, it tries the next one. It does like that until one is applicable and then stops.